metricshub-doc/src/site/markdown/troubleshooting/cli.md

```diff
@@ -104,6 +105,16 @@ This command will connect to the `CISC03` SAN switch (`sto`rage) using `SNMP` ve
 
 > Note: SAN switches fall into the **Storage** category of systems, like disk arrays and tape libraries, and not in the ~~Network~~ category, like network switches and IP routers.
 
+### Storage System, SNMP v3
+
+```batch
+$ metricshub netapp9-san -t storage --snmpv3 --snmpv3-auth SHA --snmpv3-username USERA --snmpv3-password MySECRET --snmpv3-privacy --snmpv3-retryIntervals 5,10,15 DES --snmpv3-privacy-password MyPrivacySECRET
+```
+
+This command will connect to the `netapp9-san` SAN switch (`sto`rage) using `SNMP` version `3`.
+
+> Note: If no SNMP V3 community is specified, **public** is assumed by default.

```

Comment:
`Note: If no SNMP version 3 community is specified, **public** is assumed by default.`